File: 2924/Solution.go
func findChampion(n int, edges [][]int) int {
    a := make([]int, n)
    for _, r := range edges {
        a[r[1]]++
    }
    res := -1
    for i, n := range a {
        if n == 0 {
            if res != -1 {
                return -1
            }
            res = i
        }
    }
    return res
}

File: 2924/Solution.java
class Solution {
    public int findChampion(int n, int[][] edges) {
        int[] a = new int[n];
        for(int[] r : edges){
            a[r[1]]++;
        }
        int res = -1;
        for (int i = 0; i < n; i++){
            if(a[i] == 0){
                if(res != -1) return -1;
                res = i;
            }
        }
        return res;
    }
}
